xml/System.Runtime.InteropServices/Marshal.xml

```diff
@@ -3362,7 +3362,7 @@
       <Docs>
         <param name="errorCode">The HRESULT to be converted.</param>
         <summary>Converts the specified HRESULT error code to a corresponding <see cref="T:System.Exception" /> object.</summary>
-        <returns>An object that represents the converted HRESULT.</returns>
+        <returns>An object that represents the converted HRESULT, or <see langword="null" /> if an HRESULT value, such as <see langword="S_OK" /> or <see langword="S_FALSE" />, does not represent an error code.</returns>
```

Comment:
Wondering if this flow would work better

```suggestion
        <returns>An object that represents the converted HRESULT, or <see langword="null" /> if the HRESULT value doesn't represent an error code (for example, <see langword="S_OK" /> or <see langword="S_FALSE" />) .</returns>
```

Reply:
This does work much better, @mairaw. Thanks.